src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        for (int i = 0;i < UserService.getUsers().length;i++) {
+            if (Objects.equals(UserService.getUsers()[i].getEmail(), email)) {
+                if (Objects.equals(UserService.getUsers()[i].getPassword(), password)) {
+                    return true;
+                }
+            }
+        }
         return false;
```

Comment:
I would strongly recommend to use `userService.findByEmail` for fetching user by email. Then, you can simply check password against fetched user.